src/video-player.js

```diff
@@ -112,6 +113,7 @@ class VideoPlayer extends Utils.mixin(Eventable) {
       return;
     }
     try {
+      const internalAnalyticsMetadata = this.options.videojsOptions._internalAnalyticsMetadata || {};
```

Comment:
```suggestion
      const internalAnalyticsMetadata = this.options.videojsOptions._internalAnalyticsMetadata ?? {};
```

Reply:
does not also make sense that something that is Cloudinary related is stored under `videojsOptions` ?

Reply:
good point, I moved it to playerOptions as its internal property not from videojs